site/en/docs/devtools/recorder/reference/index.md

```diff
@@ -154,6 +154,22 @@ By default, the **Recorder** replays the user flow as fast as it can. To underst
 You can use these slow replay options only in the **Recorder**. To add timeouts to the recording itself, see [Adjust timeouts for steps](/docs/devtools/recorder/reference/#adjust-timeout).
 {% endAside %}
 
+### Inspect code {: #inspect-code }
+
+To inspect the code of a user flow in various formats:
+
+1. Open a recording in the **Recorder** panel.
+1. Click **Show code** in the top right corner of the steps list.
+   {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/KhynxMityVxQeMJuNdU9.png", alt="The Show code button.", width="800", height="669" %}
+1. The **Recorder** shows a side-by-side view of the steps and their code. As you hover over a step, the **Recorder** highlights its respective code.
```

Comment:
Move to another step, show a video to illustrate the point if possible. 

"As you hover over a step, the **Recorder** highlights its respective code."